README.md

```diff
@@ -1,2 +1,53 @@
-# Portfolio
-My portfolio
+# Project Name
+Portfolio: set up and mobile first
+
+
+## Built With
+HTML
+CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
```

Comment:
- Hi @Hazard10-eng, it is a core requirement that you use the newly approved ReadMe file. Please refer to the previous reviewer's comment on this. Kindly use this new [README TEMPLATE](https://github.com/microverseinc/readme-template/blob/master/README.md).